src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/util/CliSyntax.java

```diff
@@ -5,6 +5,7 @@
  */
 public class CliSyntax {
     /* Prefix definitions */
+    public static final Prefix PREFIX_CLIENT = new Prefix("c/");
```

Comment:
Do you think this is better or not?
```suggestion
    public static final Prefix PREFIX_CLIENT_INDEX = new Prefix("ci/");
```

Reply:
The actual client index prefix should remain as c/ as we are unable to use ri/INDEX for recipe since ri is reserved for recipe ingredients

Reply:
Right, thanks